src/si/dynamic_viscosity.rs

```diff
@@ -0,0 +1,98 @@
+        // poise = 0.1 Pa · s
+        @poise: 1.0_E-1; "P",
+            "poise", "poises";
+        // centipoise = 1 mPa · s    
+        @centipoise: prefix!(centi) * 1.0_E-1; "cP",
+            "centipoise", "centipoises";  
+        @pound_force_second_per_square_foot: 4.448_222_E0 / 3.048_E-1 / 3.048_E-1; "lbf·s/ft²",
+            "pound-force second per square foot", "pound-force seconds per square foot";          
+        @pound_force_second_per_square_inch: 4.448_222_E0 / 2.54_E-2 / 2.54_E-2; "lbf·s/in²",
+            "pound-force second per square inch", "pound-force seconds per square inch";
+        // Reyn = 1 lbf·s/in²
+        @reyn: 4.448_222_E0 / 2.54_E-2 / 2.54_E-2; "reyn",
+            "reyn", "reyn";
+        @pound_per_foot_second: 4.535_924_E-1 / 3.048_E-1; "lb/(ft·s)",
+            "pound per foot second", "pounds per foot second";      
+        @pound_per_foot_hour: 4.535_924_E-1 / 3.048_E-1 / 3600.0; "lb/(ft·h)",
+            "pound per foot hour", "pounds per foot hour";
+        @slug_per_foot_second: 1.459_390_E1 / 3.048_E-1; "slug/(ft·s)",
+            "slug per foot second", "slugs per foot second";  
+        @gram_per_centimeter_second: prefix!(milli) / prefix!(centi); "g/(cm·s)",
+            "gram per centimeter second", "grams per centimeter second";
+    }
+}
+
+#[cfg(test)]
+mod test {
+    storage_types! {
+        use crate::num::One;
+        use crate::si::quantities::*;
+        use crate::si::dynamic_viscosity as dv;
+        use crate::si::time as t;
+        use crate::si::mass as m;
+        use crate::si::length as l;
+        use crate::si::pressure as p;
+        use crate::tests::Test;
+
+        #[test]
+        fn check_dimension() {
+            let _: DynamicViscosity<V> = Pressure::new::<p::pascal>(V::one())
+                * Time::new::<t::second>(V::one());
+        }
+
+        #[test]
+        fn check_units() {
+            test::<dv::pascal_second, t::second, p::pascal>();
+            test::<dv::millipascal_second, t::second, p::millipascal>();
+            test::<dv::micropascal_second, t::second, p::micropascal>();
+            test::<dv::poise, t::second, p::dyne_per_square_centimeter>();
+            test::<dv::centipoise, t::second, p::millipascal>();
+            test::<dv::pound_force_second_per_square_foot, t::second, p::pound_force_per_square_foot>();
+            test::<dv::pound_force_second_per_square_inch, t::second, p::pound_force_per_square_inch>();
+            test::<dv::reyn, t::second, p::pound_force_per_square_inch>();
+
+            fn test<DV: dv::Conversion<V>, T: t::Conversion<V>, P: p::Conversion<V>>() {
+                Test::assert_approx_eq(&DynamicViscosity::new::<DV>(V::one()),
+                    &(Pressure::new::<P>(V::one())
+                        * Time::new::<T>(V::one())));
+            }
+        }
+
+        #[test]
+        fn check_units_mlt() {
+            test::<dv::pound_per_foot_second, t::second, m::pound, l::foot>();
```

Comment:
Generic parameters in formula order again.

```suggestion
            test::<m::pound, l::foot, t::second, dv::pound_per_foot_second>();
```